Integrations/AbuseDB/AbuseDB.yml

```diff
@@ -0,0 +1,282 @@
+      type: Unknown
+    - contextPath: AbuseIPDB.IP.Geo.Country
+      description: Country associated with this IP Address
+      type: Unknown
+    - contextPath: AbuseIPDB.IP.Reports
+      description: Reports summary (for "verbose" reports)
+      type: Unknown
+    - contextPath: DBotScore.Score
+      description: The actual score.
+      type: Number
+    - contextPath: DBotScore.Vendor
+      description: Vendor used to calculate the score.
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: String
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: String
+    - contextPath: AbuseIPDB.IP.Malicious.Vendor
+      description: Vendor used to calculate the score.
+      type: String
+    - contextPath: AbuseIPDB.IP.Malicious.Detections
+      description: The Detections that led to the verdict
+      type: String
+    - contextPath: IP.Malicious.Description
+      description: A description explaining why the IP address was reported as malicious.
+      type: String
+  - arguments:
+    - default: false
+      description: ' The IP address to report'
+      isArray: false
+      name: ip
+      required: true
+      secret: false
+    - default: false
+      description: CSV list of category IDs (numerical representation or in their
+        name)
+      isArray: false
+      name: categories
+      required: true
+      secret: false
+    deprecated: false
+    description: Report an IP address to AbuseIPDB
+    execution: false
+    name: abuseipdb-report-ip
+  - arguments:
+    - default: false
+      defaultValue: '30'
+      description: Time range to return reports for (in days), default is 30
+      isArray: false
+      name: days
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '50'
+      description: 'Maximum number of IPs to retrieve, default is 50 '
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'false'
+      description: Save a list of blacklisted IPs into the Context
+      isArray: false
+      name: saveToContext
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    deprecated: false
+    description: Returns a list of the most reported IP addresses
+    execution: false
+    name: abuseipdb-get-blacklist
+    outputs:
+    - contextPath: AbuseIPDB.Blacklist
+      description: List of blacklisted IPs
+      type: Unknown
+  - deprecated: false
+    description: Returns a list of report categories from AbuseIPDB
+    execution: false
+    name: abuseipdb-get-categories
+    outputs:
+    - contextPath: AbuseIPDB.Categories
+      description: List of AbuseIPDB categories
+      type: string
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
```

Comment:
not sure it is python3, looks more like 2

Reply:
How do you know?
I usually look for the " Exception as e" and by that I know

Reply:
`Exception as e` works in both versions.
it looks like an old integration (before we switched to python3